src/main/java/htsjdk/samtools/cram/encoding/ByteArrayStopEncoding.java

```diff
@@ -1,132 +0,0 @@
-/**
```

Comment:
Moved to `external` sub-package: it only operates on external data blocks, despite not having the `EXTERNAL` encoding ID.